samples/subsys/usb/webusb/src/webusb_serial.c

```diff
@@ -287,35 +288,35 @@ static void webusb_serial_dev_status_cb(enum usb_dc_status_code status,
 {
 	ARG_UNUSED(param);
 
-	SYS_LOG_DBG("");
+	LOG_DBG("");
```

Comment:
I do not think those debugs are useful without function names, probably we should use USB_DBG()